cumulusci/tasks/marketing_cloud/deploy.py

```diff
@@ -98,14 +101,29 @@ def _poll_action(self):
         response = requests.get(
             f"{self.endpoint}/deployments/{self.job_id}", headers=self.headers
         )
-        result = safe_json_from_response(response)
-        self.logger.info(f"Waiting [{result['status']}]...")
-        if result["status"] == DEPLOY_FINISHED_STATUS:
+        response_data = safe_json_from_response(response)
+        deploy_status = response_data["status"]
+        self.logger.info(f"Deployment status is: {deploy_status}")
+
+        if deploy_status != IN_PROGRESS_STATUS:
```

Comment:
Instead of looking for completed or errored statuses to determine when a job is done, we now consider a deployment "finished" when the status changes from `IN_PROGRESS_STATUS`